Dockerfile

```diff
@@ -1,6 +1,10 @@
 FROM scratch
 
-LABEL maintainer="username"
+LABEL maintainer="alexschomb"
 
 # copy local files
 COPY root/ /
+
+# install translations
+RUN chmod +x /install-translations.sh
+CMD /install-translations.sh
```

Comment:
these two lines won't work here as it needs to be a single layer image, so only the `COPY` line is allowed in the final image